### PR TITLE
245 move events serverside

### PR DESCRIPTION
**Pull request by alison985**
#245

tests pass

### CODE REVIEW

File: client/app/pages/admin/outdated-queries/index.js [hunk 1]
```diff
@@ -4,7 +4,6 @@ import { Paginator } from '../../../utils';
 import template from './outdated-queries.html';
 
 function OutdatedQueriesCtrl($scope, Events, $http, $timeout) {
```
Comment (washort) [inline client/app/pages/admin/outdated-queries/index.js]: can Events come out of here too?

File: client/app/pages/alerts-list/index.js [hunk 2]
```diff
@@ -9,8 +9,6 @@ const stateClass = {
 
 class AlertsListCtrl {
   constructor(Events, Alert) {
```
Comment (washort) [inline client/app/pages/alerts-list/index.js]: `Events` unused?

File: client/app/pages/groups/list.js [hunk 3]
```diff
@@ -2,7 +2,6 @@ import { Paginator } from '../../utils';
 import template from './list.html';
 
 function GroupsCtrl($scope, $uibModal, currentUser, Events, Group) {
```
Comment (washort) [inline client/app/pages/groups/list.js]: `Events` unused?

**Comment by alison985** [comment]
@arikfr Updated any `'object_type': 'page'` in the redash/handlers folder to `'object_type': 'api_call'` since the page or the api is making an API call.

**Pull request merged**